doc/sql_guide.md

```diff
@@ -0,0 +1,291 @@
+# Test Objective
+Structured Query Language (SQL), SQL is an ANSI (American National Standards Institute) standard computer language. Its functions include data query, data manipulation, data definition and data control. It is a general relational database language with strong functions. It is now the standard language for relational databases. (such as SELECT, UPDATE, DELETE, INSERT, WHERE and etc)   
+CUBRID SQL test, SQL detection for each build to detect the basic functions of the database and ensure the correct output of commands.
+  
+# General SQL Test  
+## Test deployment  
+* Install CTP: [CTP_install_guide](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md)  
+    * sql test  
+    touch ~/CTP/conf/sql_local.conf    
```

Comment:
As is:
'touch ~/CTP/conf/sql_local.conf' 

To be:
Prepare test configuration file ~/CTP/conf/sql.conf

Note: do not use sql_local.conf as file name. Just adopt sql.conf